Global replace Pivotal with Tanzu, TAS, TAS Apps Manager, etc. (#255)
SteeltoeOSS/Documentation: api/v2/connectors/redis.md

cf restage myApp
```

> >NOTE: The preceding commands assume you use the Redis service provided by TAS. If you use a different service, you have to adjust the `create-service` command to fit your environment.

Reviewer: ```suggestion
>NOTE: The preceding commands assume you use [Redis for VMware Tanzu Application Service](https://docs.pivotal.io/redis/release.html). If you use a different service, you have to adjust the `create-service` command to fit your environment.
```

Side note: this link has no version number but does redirect to the latest, we should test dropping versions from links where we can (Like TAS docs) for easier maintenance
